Fix ``test_utils.TestClient`` on Python 3.11.+.
